modules/registry-configuring-storage-aws-user-infra.adoc

```diff
@@ -2,6 +2,7 @@
 //
 //* architecture/installation-.adoc
 // * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * registry/configuring-registry-storage-aws-user-infrastructure.adoc
 
 [id="registry-configuring-storage-aws-user-infra_{context}"]
 = Configuring registry storage for AWS with user-provisioned infrastructure
```

Comment:
s/an S3 bucket/a S3 bucket  i
in the following field.

Reply:
@xiuwang Where should the s/in the following field go? I'm sorry to ask, but for some reason the comment is showing up in a different place in my Git view.

Reply:
This issue is resolved in https://github.com/openshift/openshift-docs/pull/16593/files#diff-f981c0ddf4f00dc6b26823c9f04d898cL14